packages/cover-traffic-daemon/src/index.spec.ts

```diff
@@ -97,9 +110,12 @@ describe('cover-traffic daemon', async function () {
     sinon.restore()
   })
   it('should run and stop properly', async function () {
-    log('starting stubbed cover-traffic daemon')
+    log('starting stubbed hopr node')
     await node.start()
-    log('completed stubbed cover-traffic')
+    log('completed stubbed hopr node, starting cover-traffic strategy')
+    node.setChannelStrategy(new CoverTrafficStrategy(PublicKey.fromPeerId(peerId), node, data))
+    log('completed strategy, waiting for 200 ms w/o crashing')
```

Comment:
What exactly is this trying to catch?

Reply:
Any instantiating errors (i.e. `new CoverTrafficStrategy`) when adding the strategy w/o allowing an initial `tick` to happen.